src/dataframe/dataframe.jl

```diff
@@ -612,7 +612,8 @@ If `ind` is omitted it is set to `ncol(df)+1`
 
 # Arguments
 - `df` : the DataFrame to which we want to add columns
-- `ind` : a position at which we want to insert a column
+- `ref` : a position at which we want to insert a column, passed as an integer
```

Comment:
line 610 still says `ind`.
Also we might want to mention something about being inserted to the left of `ref` or that the column currently at positions `ref` is shifted to the right.

Reply:
ok - I have proposed an improvement in the docs